chitchat_server/test_workflow.py

```diff
@@ -0,0 +1,6 @@
+def test_workflow():
```

Comment:
This is important since this is probably one of the reasons that the actions are gone on github